frontend/src/metabase/nav/components/AppBar/AppBarLogo.styled.tsx

```diff
@@ -9,8 +9,11 @@ export const LogoLink = styled(Link)<{ isSmallAppBar: boolean }>`
   align-items: center;
   justify-content: center;
   border-radius: 0.375rem;
-  width: 2.25rem;
   height: 3.25rem;
+  min-width: 2.25rem;
+  max-width: 12rem;
+  overflow-x: hidden;
+  line-height: 0;
```

Comment:
Adding `line-height: 0` keeps the logo vertically aligned with the hamburger menu to its left